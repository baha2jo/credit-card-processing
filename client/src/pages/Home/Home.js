import React from 'react';
import { CardForm } from './components/CardForm';
import { CardsList } from './components/CardsList';
import './Home.scss';

export const Home = () => {
  return (
    <div className='container home h-100 d-flex justify-content-center align-items-center'>
      <div className='card h-75 w-100'>
        <div className='card-body p-0 h-100'>
          <div className='row m-0 h-100'>
            <div className='col home__cc-form h-100'>
              <CardForm />
            </div>
            <div className='col home__cc-list h-100'>
              <CardsList />
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};
