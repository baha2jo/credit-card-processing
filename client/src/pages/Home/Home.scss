@import 'src/assets/styles/colors';

$border-radius: 10px;

.home {
  .card {
    border-radius: $border-radius;
    box-shadow: 0px 15px 50px 10px rgba(0, 0, 0, 0.2);
  }
  &__cc-form {
    background-color: $blue;
    border-radius: $border-radius 0 0 $border-radius;
  }
  &__cc-list {
    border-radius: $border-radius 0 0 $border-radius;
  }
}
