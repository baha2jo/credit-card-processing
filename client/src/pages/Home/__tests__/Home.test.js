import { Home } from '../Home';

describe('Home Page', () => {
  it('should render correctly', () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
