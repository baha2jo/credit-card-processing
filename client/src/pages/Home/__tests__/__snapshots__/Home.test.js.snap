// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render correctly 1`] = `
<div
  className="container home h-100 d-flex justify-content-center"
>
  <div
    className="card h-75 w-100 mt-5"
  >
    <div
      className="card-body p-0 h-100"
    >
      <div
        className="row m-0 h-100"
      >
        <div
          className="col home__cc-form h-100 "
        >
          <CardForm />
        </div>
        <div
          className="col home__cc-list h-100 pb-5"
        >
          <CardsList />
        </div>
      </div>
    </div>
  </div>
</div>
`;
